fendermint/docker/runner.Dockerfile

```diff
@@ -30,5 +29,7 @@ RUN chmod 777 /fendermint/logs
 COPY fendermint/docker/.artifacts/bundle.car $FM_HOME_DIR/bundle.car
 COPY fendermint/docker/.artifacts/custom_actors_bundle.car $FM_HOME_DIR/custom_actors_bundle.car
 COPY fendermint/docker/.artifacts/contracts $FM_HOME_DIR/contracts
+COPY fendermint/docker/docker_entry.sh /usr/local/bin/docker_entry.sh
```

Comment:
nit: I think kebab-casing is more idiomatic with scripts, ie. `docker-entry.sh`

Reply:
Sure, fixed!